lighthouse-core/audits/content-width.js

```diff
@@ -38,17 +38,9 @@ class ContentWidth extends Audit {
    * @return {!AuditResult}
    */
   static audit(artifacts) {
-    if (typeof artifacts.ContentWidth === 'undefined' ||
-        typeof artifacts.ContentWidth.scrollWidth === 'undefined' ||
```

Comment:
yeah, some of these I was probably a little too eager to rip out nearby useless stuff :)

This one won't be caught by artifact shape check, though, because it'll still be the content of the artifact (which could conceivably have an undefined `scrollWidth`). Just annoys me because if it wants to assert that the values are numbers, assert *that*, and preferably do so in the gatherer.

I'll remove from this PR